WISE-Community/WISE-Client
# refactor(Graph): Move plot line code to its own file

## Changes

Moved plot line code to plot-line-manager.ts.

## Test

- Make sure "X Axis Plot Line" and "Y Axis Plot Line" show up (author this in Graph advanced authoring)
- Make sure "Show Mouse X Plot Line" and "Show Mouse Y Plot Line" works (author this in Graph advanced authoring)
- Make sure basic Graph functionality still works

Closes #1264
